Makefile.core.mk

```diff
@@ -175,20 +174,47 @@ docker-push-nightly: docker-build
 	docker tag ${IMAGE} $(HUB)/$(IMAGE_BASE):$(MINOR_VERSION)-latest
 	docker push $(HUB)/$(IMAGE_BASE):$(MINOR_VERSION)-latest
 
+# NIGHTLY defines if the nightly image should be pushed or not
+NIGHTLY ?= false
+
+# BUILDX_OUTPUT defines the buildx output
+# --load builds locally the container image
+# --push builds and pushes the container image to a registry
+BUILDX_OUTPUT ?= --push
+
+# BUILDX_TAGS are the --tag flag passed to the docker buildx build command.
+BUILDX_TAGS = --tag ${IMAGE}
+ifeq ($(NIGHTLY),true)
+BUILDX_TAGS += --tag $(HUB)/$(IMAGE_BASE):$(MINOR_VERSION)-nightly-$(TODAY)
+endif
+
 # PLATFORMS defines the target platforms for  the manager image be build to provide support to multiple
 # architectures. (i.e. make docker-buildx IMAGE=myregistry/mypoperator:0.0.1). To use this option you need to:
 # - able to use docker buildx . More info: https://docs.docker.com/build/buildx/
 # - have enable BuildKit, More info: https://docs.docker.com/develop/develop-images/build_enhancements/
 # - be able to push the image for your registry (i.e. if you do not inform a valid value via IMAGE=<myregistry/image:<tag>> then the export will fail)
 # To properly provided solutions that supports more than one platform you should use this option.
 PLATFORMS ?= linux/arm64,linux/amd64,linux/s390x,linux/ppc64le
```

Comment:
@jwendell By default, it builds for these four architectures (arm64, amd64, s390x and ppc64le). I know that the first target platforms are `linux/amd64` and `linux/arm64`. Do you think we leave it at is or we should remove `linux/s390x` and `linux/ppc64le` for now?

Reply:
"by default" you mean if we run `make docker-buildx` or `make build-all`, right?

if I run `make build` it will build only linux/am64, correct?

Reply:
Correct:
- `make build` builds the manager binary for `linux/amd64` platform only
- `make build-all` builds the manager binaries for the listed platforms
- `make docker-buildx` builds and pushes the multi-platform container image